CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -1097,6 +1097,38 @@ static function ValidateXpMultiplers ()
 	}
 }
 
+static function GrantBonusStartUpStaff (XComGameState StartState)
```

Comment:
This function should be in `X2DownloadableContentInfo_CovertInfiltration` since it's only ever used from one place